merlin/merlin.py

```diff
@@ -160,18 +161,24 @@ def generate_analysis_tasks_and_snakefile(dataSet: dataset.MERFISHDataSet,
 
 def run_with_snakemake(
         dataSet: dataset.MERFISHDataSet, snakefilePath: str, coreCount: int,
-        snakemakeParameters: Dict = {}, report: bool = True):
+        snakemakeParameters: Dict = {}, reportPath: str = None):
     print('Running MERlin pipeline through snakemake')
+
+    if 'restart_times' not in snakemakeParameters:
+        snakemakeParameters['restart_times'] = 3
```

Comment:
I worry that the typical user would not expect this behavior and might not understand why they have one job restarting itself. Could we log this somewhere the user might expect to find this type of information?